addons/project/tests/test_access_rights.py

```diff
@@ -129,7 +129,8 @@ def test_task_allowed_portal_read(self):
         self.project_pigs.message_subscribe(partner_ids=[self.env.user.partner_id.id])
         self.task.flush_model()
         self.task.invalidate_model()
-        self.task.with_user(self.env.user).name
+        with self.assertRaises(AccessError, msg="%s should not be able to read the task" % self.env.user.name):
```

Comment:
```suggestion
        with self.assertRaises(AccessError, msg=f"{self.env.user.name} should not be able to read the task"):
```
We can use a fstring here